src/main/java/emu/grasscutter/game/props/EntityType.java

```diff
@@ -75,6 +75,7 @@ public enum EntityType implements IntValueEnum {
     Screen(64),
     EchoShell(65),
     UIInteractGadget(66),
+    Region(67),
```

Comment:
Best use some high value for custom stuff, since otherwise we will quickly run into collisions.
Also best add a comment that this is nothing real from official, and needs more investigation.